datumaro/plugins/vgg_face2_format.py

```diff
@@ -43,16 +57,29 @@ def _load_items(self, path):
 
         for row in landmarks_table:
             item_id = row['NAME_ID']
-            image_path = osp.join(self._dataset_dir, self._subset,
-                item_id + VggFace2Path.IMAGE_EXT)
-            annotations = []
+            label = osp.dirname(item_id)
+            if 0 < len(label):
+                item_id = osp.basename(item_id)
+
+            points = None
             if len([p for p in row if row[p] == '']) == 0 and len(row) == 11:
-                annotations.append(Points(
-                    [float(row[p]) for p in row if p != 'NAME_ID']))
-            if item_id in items and 0 < len(annotations):
+                points = Points(
+                    [eval(row[p]) for p in row if p != 'NAME_ID'])
+
+            if item_id in items:
                 annotation = items[item_id].annotations
-                annotation.append(annotations[0])
+                if points is not None:
+                    annotation.append(points)
             else:
+                image_path = osp.join(self._dataset_dir, self._subset, label,
+                    item_id + VggFace2Path.IMAGE_EXT)
+                annotations = []
+                if 0 < len(label):
+                    label = self._categories[AnnotationType.label].find(label)[0]
```

Comment:
I think, the file with labels would be useful, if it denoted the mapping between folder names and labels. If there is no file, we just use folder names. The same thing as in `ImageNet` - folders are called `n12345`, but there is a file with mapping to normal descriptive names.

Reply:
I agree, this file is not needed. If `save_images is false`, we create empty folders. But this does not work in CVAT, because empty folders are not saved in the zip archive.

Reply:
It is important notion about CVAT, but it does not contradict anything I said. Do we need to maintain empty label folders from CVAT?

What I suggested was allowing to specify label-directory mapping in these files (`ImageNet` `synsets.txt`-style), but it looks like a good candidate for `ImageNet` format extension, though.

Reply:
I don't quite understand what you are suggesting. 
`synsets.txt` in `ImageNet_txt` format is the same as here. It is needed to create `categories` correctly when we do not save images or when not all labels  are affected.
Do you propose in this file to describe the mapping of the folder name and the object name (n01234 cat)?

Reply:
> Do you propose in this file to describe the mapping of the folder name and the object name (n01234 cat)?

Yes, like this: 
https://gist.github.com/yrevar/942d3a0ac09ec9e5eb3a#gistcomment-2575723